misc/userscripts/qute-pass

```diff
@@ -44,8 +44,16 @@ import sys
 
 import tldextract
 
-argument_parser = argparse.ArgumentParser()
-argument_parser.add_argument('url', nargs='?', default=os.environ['QUTE_URL'])
+argument_parser = argparse.ArgumentParser(description=(
```

Comment:
Please make the comment above a [docstring](https://en.wikipedia.org/wiki/Docstring#Python) and use `__doc__` instead of duplicating it here.